test-cases/tests/upgrade/n02b-upgrade-rhoam.md

```diff
@@ -75,16 +62,21 @@ Note: If [N09 test case](https://github.com/integr8ly/integreatly-operator/blob/
    oc patch installplan install-<hash> --namespace redhat-rhoam-operator --type merge --patch '{"spec":{"approved":true}}'
    ```
 
-5. Poll cluster to check when the RHOAM upgrade is completed (update version to match currently tested version (e.g. `1.8.0`)):
+4. Poll cluster to check when the RHOAM upgrade is completed (update version to match currently tested version (e.g. `1.8.0`)):
 
    ```bash
-   watch -n 60 " oc get rhmi rhoam -n redhat-rhoam-operator -o json | jq -r .status.version | grep -q "2.x.x" && echo 'RHOAM Upgrade completed\!'"
+   watch -n 60 " oc get rhmi rhoam -n redhat-rhoam-operator -o json | jq -r .status.version | grep -q "1.x.x" && echo 'RHOAM Upgrade completed\!'"
    ```
 
    > This script will run every 60 seconds to check whether the RHOAM upgrade has finished
    >
    > Once it's finished, it should print out "Upgrade completed!"
 
+5. In OpenShift console, click on the "bell" icon in the top right corner. Go through the notifications that were generated in the time between the RHOAM upgrade started and ended.
+
+   > Verify that there were no RHOAM-related alerts firing during the upgrade
+   > If unsure whether the alert is RHOAM-related, consult it with engineering
+
 6. Go to the OpenShift console, go through all the `redhat-rhoam-` prefixed namespaces and verify that all routes (Networking -> Routes) of RHOAM components are accessible
```

Comment:
@psturc  is this the right thing to do or would we rather be going through the links available in UI (also worth mentioning that it might not work if you are logged in as kubeadmin?)

Reply:
I think it's still worth to check the links in services' namespaces, but I've added the step you've mentioned as well. Is it ok like that?